lvx frontend/src/Sell.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from "react";
+import "./Sell.css";
+import { BiImageAdd } from "react-icons/bi";
+function Sell() {
+  const [values, setValues] = useState({
+    seller: '',
+    itemname: '',
+    price: '',
+    whatsapp: '',
+    lpuid: '',
+    description: '',
+    image: "",
+    // unit: 'day',
+    // type: 'accessories',
+  });
+  const handleSubmit = async (e) => {
+    e.preventDefault()
+    console.log(values)
+    try {
+      const response = await fetch('http://127.0.0.1:3000/api/v1/items',{
```

Comment:
Add domain here in `. env` later when you change while deploying you have to do less changes.